### DISCUSSION TITLE
Start/Stop values

### DISCUSSION THREAD

**Discussion by Keight8** [EvanMulawski/FanControl.CorsairLink#41] [Q&A] [answered]
Hi, What start/stop, step up/down, offset values should I set  for my H115i Pro XT?
Thanks

**Comment by EvanMulawski** [comment, marked as answer]
Hi @Keight8,

For the pump, it's completely up to you as the percentage value will be mapped to the preset pump mode (see Footnote 4 in the [README](https://github.com/EvanMulawski/FanControl.CorsairLink/blob/main/README.md)).

For the fans, the start and stop values can be determined automatically by Fan Control - click the three dots menu on the control and select **Detect fan start/stop % automatically**. Note that some fans will never completely stop. The step and offset values are up to you.

See https://github.com/Rem0o/FanControl.Releases/discussions/871 for more information.
